src/jit/lower.cpp

```diff
@@ -2454,7 +2455,12 @@ void Lowering::LowerCompare(GenTree* cmp)
                 }
             }
         }
+#endif
+#ifdef _TARGET_XARCH_
         else if (op1->OperIs(GT_AND) && cmp->OperIs(GT_EQ, GT_NE))
+#else
+        if (op1->OperIs(GT_AND) && cmp->OperIs(GT_EQ, GT_NE))
+#endif
```

Comment:
I would suggest to restructure the above 6 lines as:
```
        else
#endif \\ _TARGET_XARCH_
             if (op1->OperIs(GT_AND) && cmp->OperIs(GT_EQ, GT_NE))
```

Reply:
@CarolEidt done